pkg/dashboard/store/store.go

```diff
@@ -60,8 +58,6 @@ func NewDBStore(lc fx.Lifecycle, conf *config.ChaosDashboardConfig) (*gorm.DB, e
 
 	gormDB, err := gorm.Open(conf.Database.Driver, ds)
 	if err != nil {
-		log.Error(err, "Failed to open DB: ", "driver => ", conf.Database.Driver)
-
```

Comment:
Well, you could not complete this PR by removing the logging. :P

The expected changes are:

- use the instance of logger instead of the global logger;
- you could bootstrap the logger as what does in https://github.com/chaos-mesh/chaos-mesh/pull/3035
- keep the origin printed logging

Reply:
Hey @STRRL ,Thanks for the feedback. I was under the impression that logging was not needed for the package from the title. I'll make the changes accordingly.

Reply:
Hey @STRRL, I made the changes,can you check the latest commit?

Reply:
Updated changelog!